frappe/public/js/frappe/list/list_view.js

```diff
@@ -1029,8 +1029,11 @@ frappe.views.ListView = class ListView extends frappe.views.BaseList {
 		link.dataset.doctype = this.doctype;
 		link.dataset.name = doc.name;
 		link.href = this.get_form_link(doc);
-		link.title = value.toString();
-		link.textContent = value.toString();
+		// "Text Editor" and some other fieldtypes can have html tags in them so strip and show text.
+		// If no text is found show "No Text Found in {Field Label}"
+		let textValue = frappe.utils.html2text(value) || "No Text Found in " + subject_field.label;
```

Comment:
Let's keep it empty if no text is found instead of "No Text Found in" since it'll be hard to distinguish among other text.

Reply:
leaving it blank will be confusing to users on where to click?